lib/web_ui/dev/utils.dart

```diff
@@ -219,7 +219,7 @@ void cleanup() async {
     }
   }
 
-  cleanupCallbacks.forEach((element) {
-    element.call();
+  cleanupCallbacks.forEach((element) async {
+    await element.call();
```

Comment:
What's the effect of this change?

Reply:
When I  didn't run in like this, I wasn't able to see the INFO, I print out: 
"print('INFO: Simulator ${iosSimulator.id} shutdown.');"

The previous implementation was missing the `await`. Let me know if I'm missing something. I'm open to suggestions on change.

Reply:
Hmm, strange. This shouldn't have any effect because `cleanupCallbacks.forEach` is synchronous anyway; it ignores `async` and `await`. However, the following would have an effect:

```
for (AsyncCallback cleanupCallback in cleanupCallbacks) {
  await cleanupCallback();
}
```

This is because the `for` loop is aware of `await` and it would not pick up the next element until the previous callback's future is resolved.

Reply:
This worked when I also combined it with adding a `await` to the call inside test_runner.dart 

thanks for the suggestions.